packages/components/pie-divider/test/visual/pie-divider.spec.ts

```diff
@@ -2,13 +2,24 @@
 import { test } from '@sand4rt/experimental-ct-web';
 import percySnapshot from '@percy/playwright';
 import { PieDivider, DividerProps } from '@/index';
+import { variants, orientations } from '@/defs';
 
-test.describe('PieDivider - Visual tests`', () => {
-    test('should display the PieDivider component successfully', async ({ page, mount }) => {
+variants.forEach((variant) => {
+    test(`should render correctly with variant = ${variant}`, async ({ page, mount }) => {
         await mount(PieDivider, {
-            props: {} as DividerProps,
+            props: { variant } as DividerProps,
         });
 
-        await percySnapshot(page, 'PieDivider - Visual Test');
+        await percySnapshot(page, `Divider - variant = ${variant}`);
+    });
+});
+
+orientations.forEach((orientation) => {
+    test(`should render correctly with orientation = ${orientation}`, async ({ page, mount }) => {
+        await mount(PieDivider, {
+            props: { orientation } as DividerProps,
+        });
+
+        await percySnapshot(page, `Divider - orientation = ${orientation}`);
```

Comment:
I would copy what we do for pie-button and render all of these variations in the same test. I don't think it will get unruly and it'll speed up these tests and reduce percy usage :)

Reply:
good idea, will add it!